Dockerfiles/mlpack-dependencies-alpine/Dockerfile

```diff
@@ -0,0 +1,50 @@
+#Using ubuntu's latest image as base-image
+FROM debian:latest
```

Comment:
This one isn't based on alpine, I guess maybe it is better to just remove this Dockerfile since we can't use it.